CHANGELOG

```diff
@@ -1,13 +1,15 @@
 devel
 -----
 
+* Taken collection dropping from fast track in maintenance
```

Comment:
```suggestion
* Taken collection dropping from fast track in maintenance.
```

Reply:
It would be awesome if there was a description here why this change is necessary, i.e. which problem it fixes. Otherwise end users simply cannot understand this changelog entry.

Reply:
Done